doc/source/index.rst

```diff
@@ -4,7 +4,7 @@
 
 .. |title| replace:: bg-mpl-stylesheets documentation
 
-bg-mpl-stylesheets - A package for using Billinge group style files.
+bg-mpl-stylesheets - A package for using Billinge group style files..
```

Comment:
extra period?

Reply:
also, while we are at it, let's say "Matplotlib style files" instead of just style files

Reply:
done